ocs_ci/utility/performance_dashboard.py

```diff
@@ -39,21 +38,21 @@ def initialize_data():
     Returns:
         dict: A dictionary contains the data to push to the dashboard
     """
-    csv = ocp.OCP(kind='csv', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
     worker_type = get_typed_nodes(num_of_nodes=1)[0].data['metadata'][
         'labels'
     ]['beta.kubernetes.io/instance-type']
 
-    csv_vers = csv.get()['items'][0]['spec']['version'][:-3].split("-")
-    min_version = csv_vers[0]
-    build_id = csv_vers[1]
-    ocs_ver = ".".join(min_version.split(".")[:-1])
+    (ocs_ver_info, _) = get_ocs_version()
+    ocs_ver_full = ocs_ver_info['status']['desired']['version']
+    m = re.match(r"(\d.\d).(\d)-", ocs_ver_full)
+    if m.group(1) is not None:
+        ocs_ver = m.group(1)
```

Comment:
If we do not enter this `if` block `ocs_ver` is never initialized and we will hit an error when using this variable below (lines 54 and 56). We should instead raise an error with a clear message if we are unable to initialize `ocs_ver`.

Reply:
It is intentional so that the test can fail and we can check why the version is mismatched and take corrective steps if it fails.

Reply:
I agree that we should fail, but we should raise an informative error message (referring to us being unable to retrieve the ocs version) instead of the user hitting `NameError ocs_ver is not defined` or something similar. This can be taken up in another PR as well, but I think it's worth doing.